src/newSeed/stage5.ts

```diff
@@ -1,5 +1,34 @@
 import { BaseStage } from "newSeed/baseStage";
+import { Utils } from "services/utils";
 
 export class Stage5 extends BaseStage {
-
+  submit(): void {
+    let message: string;
+    // Validate current stage
+    // TODO: Check if there is an email validator
+    if (!Utils.isValidEmail(this.seedConfig.contactDetails.contactEmail)) {
+      message = "Please enter a value for Contact Email";
+    } else if (!this.seedConfig.contactDetails.remarks) {
+      message = "Please enter a value for Additional Remarks";
+    } else if (!Utils.isValidFile(this.seedConfig.contactDetails.logo)) {
```

Comment:
Please put this method in the viewmodel, not in Utils.

We also need to validate that it is a valid URL.